packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -57,8 +57,9 @@ export type JsonProject = {
 export type JsonSuite = {
   title: string;
   location?: JsonLocation;
-  suites: JsonSuite[];
-  tests: JsonTestCase[];
+  entries: (JsonSuite | JsonTestCase)[];
```

Comment:
You would need to make entries optional if you want to support old reports in the merger.